### DISCUSSION TITLE
How to fix broken symbols in PUTTy and other win apps?

### DISCUSSION THREAD

**Discussion by ammnt** [starship/starship#2788] [Q&A]
Hello again guys!

Someone know how to fix these broken images in Putty and Microsoft Store's linux apps?

![2021-06-07_11-38-48](https://user-images.githubusercontent.com/45385632/121001121-9e9ad800-c793-11eb-8ac5-9f8b0088f64a.png)

I usally use FiraCode Nerd Font Mono and tried to change to another version but no success. I like mono fonts actually and it works fine with my macOS and Debian remote SSH. Please, advise! PS.:

```
starship 0.51.0
tag:v0.51.0
commit_hash:88c3844d
build_time:2021-03-23 16:36:25
build_env:rustc 1.50.0 (cb75ad5db 2021-02-10),
```

Thank you.
Cheers😱

**Comment by ammnt** [comment]
No one know how to fix it?🤔

**Reply by matchai** [reply]
I'd recommend trying the suggested fixes in this FAQ entry:
https://starship.rs/faq/#why-don-t-i-see-a-glyph-symbol-in-my-prompt

**Comment by ammnt** [comment]
@matchai , thank you for your answer, but I already tried it and no success. There's some problem with the font. I tried many fonts from the Firacode bandle, but everything remained the same😪
